bundles/org.openhab.binding.meross/src/main/java/org/openhab/binding/meross/internal/handler/MerossLightHandler.java

```diff
@@ -12,155 +12,54 @@
-        }
-    }
-
-    public void initializeBridge(ThingStatus bridgeStatus) {
-        if (bridgeStatus == ThingStatus.ONLINE) {
-            updateStatus(ThingStatus.ONLINE);
-        } else {
-            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE);
-        }
-    }
-
-    public void initializeThing(int lightStatus) {
-        if (lightStatus == MerossEnum.OnlineStatus.UNKNOWN.value()
-                || lightStatus == MerossEnum.OnlineStatus.NOT_ONLINE.value()
-                || lightStatus == MerossEnum.OnlineStatus.UPGRADING.value()) {
-            updateStatus(ThingStatus.UNKNOWN);
-        } else if (lightStatus == MerossEnum.OnlineStatus.OFFLINE.value()) {
-            updateStatus(ThingStatus.OFFLINE);
-        } else if (lightStatus == MerossEnum.OnlineStatus.ONLINE.value()) {
-            updateStatus(ThingStatus.ONLINE);
-        }
-    }
-
-    @Override
-    public void handleCommand(ChannelUID channelUID, Command command) {
-        if (thing.getStatus() == ThingStatus.OFFLINE) {
-            updateStatus(ThingStatus.OFFLINE);
-            return;
-        }
-        MerossBridgeHandler merossBridgeHandler = this.merossBridgeHandler;
-        if (merossBridgeHandler == null) {
-            return;
-        }
-        var merossHttpConnector = merossBridgeHandler.getMerossHttpConnector();
-        MerossManager merossManager = null;
-        if (merossHttpConnector != null) {
-            merossManager = MerossManager.newMerossManager(merossHttpConnector);
-        }
-        if (channelUID.getId().equals(CHANNEL_POWER)) {
-            if (command instanceof OnOffType) {
-                try {
-                    if (OnOffType.ON.equals(command)) {
-                        if (merossManager != null) {
-                            merossManager.sendCommand(config.lightName, MerossEnum.Namespace.CONTROL_TOGGLEX.name(),
-                                    OnOffType.ON.name());
-                        }
-                    } else if (OnOffType.OFF.equals(command)) {
-                        if (merossManager != null) {
-                            merossManager.sendCommand(config.lightName, MerossEnum.Namespace.CONTROL_TOGGLEX.name(),
-                                    OnOffType.OFF.name());
-                        }
-                    }
-                } catch (IOException | MerossMqttConnackException e) {
-                    updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR,
-                            "Cannot send command" + e.getMessage());
-                }
-            } else if (command instanceof RefreshType) {
-                logger.debug("Refresh command not supported");
-            } else {
-                logger.debug("Unsupported command {} for channel {}", command, channelUID);
-            }
-        } else {
-            logger.debug("Unsupported channelUID {}", channelUID);
-        }
+        super.initialize();
     }
 }
```

Comment:
I do not see MerossLightHandler#handleCommand implementation anymore as far as CHANNEL_POWER channel.

Reply:
Unless I made a mistake, it is handeld in the super class. Actually, the only reason at the moment to have a specific light handler class is for the change in parameter lightName. The conversion is done automatically.